### PR TITLE
Week2

### PR DESCRIPTION
**Pull request by majidkhan**

### CODE REVIEW

File: week2/homework.py [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+
+print('\n ## Task 5 ## \n')
+# Write a Python script to check whether a given key already exists in a dictionary.
+
+jet_dict = {'name': 'F-35', 'country': 'USA', 'date': 2006}
+
+
+def is_key_exist(jet_dict, key):
+
+    if key in jet_dict.keys():
```
Comment (Rassibassi) [inline week2/homework.py:67]: good job!
also just `if key in jet_dict:` works

File: week2/homework.py [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+# Additional Exercises:
+
+print('\n ## Task 6 ## \n')
+# Write a function called showNumbers that takes a parameter called limit. It should print all the numbers between 0 and limit with a label to identify the even and odd numbers.
+
+
+def show_numbers(limit):
+    for i in range(limit):
+        msg = "EVEN" if i % 2 == 0 else "ODD"
+        print(i, msg)
```
Comment (Rassibassi) [inline week2/homework.py:91]: nicely done

**Review by Rassibassi — COMMENTED** [review]
Looks good :) see my inline comments

File: week2/homework.py (hunk 2 above)
Comment (majidkhan) [reply week2/homework.py:91]: Thanks!

**Pull request merged**